pkg/config/configenv/test.go

```diff
@@ -120,7 +120,7 @@ var TestingRequesterConfig = types.RequesterConfig{
 		ProbeExec:           "",
 	},
 	JobStore: types.StorageConfig{
-		Type: types.InMemory,
+		Type: types.BoltDB,
```

Comment:
Are we sure we want the testing defaults to go on disk?

Reply:
I think so. If we get any evidence that they slow down the tests dramatically, we can always revert back (or better still, improve the performance).  One thing I think we're likely to not see in the tests, but will in long-running request nodes, is a reduction in RAM usage.  As we're no longer holding all of the state in non-aligned structs in memory.